tests/integration_test_helpers.rs

```diff
@@ -0,0 +1,27 @@
+mod common;
+use http::StatusCode;
+
+/*
+Helpers to me include the following functions
+- get
+- http_get
+- post
+- http_post
+*/
+
+#[tokio::test]
+async fn test_http_get() {
+    let clinet = common::setup();
+    let path = "/eth/v1/builder/validators";
+    let response = clinet.http_get(path).await.unwrap();
+    assert_eq!(&response.url().as_str(), &"http://127.0.0.1:5052/eth/v1/builder/validators");
```

Comment:
I don't think we need to test the URL construction for each endpoint, we can just check if we get the data we expect